<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWindow</class>
 <widget class="QMainWindow" name="SimulationWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DLSim</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QCheckBox" name="cbWarning">
      <property name="text">
       <string>Warn when finished</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTableWidget" name="twEvents">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Instant (t)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Transmission Event</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Computer #</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Network Event</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Transmited Data (current)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Transmited Data (accumulated)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data to transmit</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btInterrupt">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btExportSettings">
        <property name="text">
         <string>Export settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btExportResults">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Export results</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QProgressBar" name="pbSimProgress">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
      <property name="format">
       <string>%p% - %v/100</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
